templates/master/00-master/azure/files/opt-libexec-openshift-azure-routes-sh.yaml

```diff
@@ -131,7 +131,11 @@ contents:
     remove_stale_routes() {
         ## find extra ovn routes
         local ovnkContainerID=($(crictl ps --name ovnkube-controller | awk '{ print $1 }' | tail -n+2))
```

Comment:
I think we should remove the outer () to not execute this in a subshell

Reply:
ack removing this, I admit I did not intentionally add those braces.